### DISCUSSION TITLE
CEST sequences for VE11P and XA61

### DISCUSSION THREAD

**Discussion by schuenke** [cest-sources/cestineers#4] [Q&A] [answered]
Hey,
is there any CEST sequence (WIP816x, snapshot-CEST, pulseq-cest hybrid) available for software versions VE11P (Biograph mMR) and/or XA61 (Cima.X)? I think both versions were not listed on the slides yesterday, but maybe there is a chance to get these anyway?

Best
Patrick

**Comment by mzaiss** [comment, marked as answer]
we might have an older version for the VE11P.
There were different baselines. you might need to find out the SP#

XA61 : not yet.
